scRNA-seq/pre-processing_scRNA-seq.md

```diff
@@ -0,0 +1,116 @@
+
+For a more extensive run through on single-cell technologies, 
+Kiselev at al have very [good tutorial for scRNA-seq](https://hemberg-lab.github.io/scRNA.seq.course/introduction-to-single-cell-rna-seq.html#experimental-methods). 
+  
+### 1) Non-Tag-Based scRNA-seq
+*Examples: Smart-seq2*  
+Cells are physically separated generally into individual wells in a plate and 
+often also sorted by other means (eg. Fluorescence Activated Cell Sorting). 
+Each cell is then sequenced individual and has it's own fastq file (this will be two fastq files if this is paired-end sequencing.) 
+The data pre-processing steps for these types of scRNA-seq data is/can be more similar to 
+bulk rna-seq methods.
+
+#### Pros:  
+- Can be paired end sequencing which has less risk for 3' bias.  
+- More complete coverage of transcripts, which may be better for transcript 
+discovery purposes.   
+  
+#### Cons:  
+- Is not very efficient (generally 96 cells per plate)  
+- Takes much longer to run.  
+- Is a lot more expensive.  
+  
+### 2) Tag-based scRNA-seq
+*Examples: 10X Genomics, Drop-seq*  
+Cells are separated by emulsion/droplets, and individual cells are given barcodes. 
+Everything is then sequenced. 
+These types of methods, because they are newer, are more likely to have Unique 
+Molecular Identifiers (UMIs) which allow you to better control for PCR amplification errors and biases.
+Individual samples have two fastq files: one for the cell barcodes
+and another with the individual reads. 
+
+#### Pros:  
+- Can run potentially millions of cells at once.   
+- A lot quicker computing wise.  
+- Won't take up all your computer's storage.  
+- A lot cheaper  
+  
+#### Cons:  
+- Sequencing is not bidirectional so data will likely have more intense 3' bias.  
+- Coverage of these technologies generally is not as deep.  
+
+### Processing scRNA-seq data
+
+#### Step 0: Obtaining the data
+10X Genomics has plenty of [datasets](https://support.10xgenomics.com/single-cell-gene-expression/datasets) 
+available to play around with. 
+For this example, we will use the [1k pbmc](https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/pbmc_1k_v2) 
+(peripheral blood mononuclear cells). 
+We will use one of these samples' subsetted fastq file of this as an example.  
+  
+*Note*: depending on the state of the data you are working with, ie. if you have 
+a `.bcl` file, you will need to use `CellRanger` with their `mkfastq` command to 
+make fastq files from this.
+However, because most publicly available data is in fastq format and any
+core you might be working with will also likely provide you with fastq files, 
+we are starting from this point.  
+
+#### Step 1: Set up your directories
+As described in a previous module, we'll make some directories from command line
+for us to store these data in. 
+``` 
+mkdir data
+mkdir ref_files
+mkdir results
+mkdir data/alevin_output
+```
+
+#### Step 2: Index the human transcriptome with Salmon
```

Comment:
I think we mentioned this in person: plan is to make the index. This index will like have to be slightly different than the bulk RNA-seq index,  because it probably should have a smaller k.